### PR TITLE
Release notes for 7.0.19 / 6.1.42 / 5.5.53

### PR DESCRIPTION
**Pull request by knisbet**

### CODE REVIEW

File: docs/7.x/changelog.md [hunk 1]
```diff
@@ -602,6 +616,20 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 6.1 Releases
 
+### 6.1.42 LTS (October 6th, 2020)
+
+#### Improvements
+* GRPC logging will be enabled when passing the `--debug` flag to gravity commands ([#2183](https://github.com/gravitational/gravity/pull/2178)).
```
Comment (r0mant) [inline docs/7.x/changelog.md:622]: PR number and link don't match.

File: docs/7.x/changelog.md [hunk 2]
```diff
@@ -602,6 +616,20 @@ to learn how to gain insight into how the cluster status changes over time.
...
+
+#### Bugfixes
+
+* Fix a security issue when using SAML authentication to an identity provider (CVE-2020-15216) ([#2193](https://github.com/gravitational/gravity/pull/2193)).
+* Fix an issue with serf clusters missing nodes when partitioned for more than 24 hours ([#2176](https://github.com/gravitational/gravity/pull/2027), [planet#759](https://github.com/gravitational/planet/pull/759)).
```
Comment (r0mant) [inline docs/7.x/changelog.md:627]: Gravity PR number and link don't match.

File: docs/7.x/changelog.md [hunk 3]
```diff
@@ -1352,6 +1380,23 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 5.5 Releases
 
+### 5.5.53 LTS (October 6th, 2020)
+
+#### Improvements
+* GRPC logging will be enabled when passing the `--debug` flag to gravity commands ([#2183](https://github.com/gravitational/gravity/pull/2178)).
```
Comment (r0mant) [inline docs/7.x/changelog.md:1386]: PR number and link don't match.

File: docs/7.x/changelog.md [hunk 4]
```diff
@@ -1352,6 +1380,23 @@ to learn how to gain insight into how the cluster status changes over time.
...
+* Gravity debug reports will include additional gravity configuration ([#2160](https://github.com/gravitational/gravity/pull/2160)).
+
+#### Bugfixes
+
+* Fix a security issue when using SAML authentication to an identity provider (CVE-2020-15216) ([#2193](https://github.com/gravitational/gravity/pull/2193)).
+* Fix an issue with the backend not using linerizable reads ([#2179](https://github.com/gravitational/gravity/pull/2179)).
+* Fix an issue with recent checks that fail during operation phases where gravity-site or etcd are unavailable ([#2164](https://github.com/gravitational/gravity/pull/2164)).
+* Fix an issue with serf clusters missing nodes when partitioned for more than 24 hours ([#2176](https://github.com/gravitational/gravity/pull/2027), [planet#759](https://github.com/gravitational/planet/pull/759)).
```
Comment (r0mant) [inline docs/7.x/changelog.md:1394]: Don't match.

**Review by r0mant — APPROVED** [review]
Noticed a few mismatched PR numbers/links.

**Comment by knisbet** [comment]
> Noticed a few mismatched PR numbers/links.

Reminds me of why pilots always restart a checklist from scratch when getting interrupted. Got interrupted part way through and missed fixing a bunch of the copy pasta issues... oops.

**Pull request merged**